src/providers/ogr/qgsogrprovider.cpp

```diff
@@ -1534,6 +1534,12 @@ bool QgsOgrProvider::changeAttributeValues( const QgsChangedAttributesMap &attr_
 
   setRelevantFields( ogrLayer, true, attributeIndexes() );
 
+  bool hasTransaction = OGR_L_TestCapability( ogrLayer, OLCTransactions ) == TRUE;
```

Comment:
Can you either replace the `TRUE` with `true` or remove it (just for consistency)